dns_pre_lda.scala

```diff
@@ -58,7 +59,7 @@ def toDouble(s: String, default:Double=Double.NaN) = {if (isNumeric(s)) s.toDoub
 
 
 val file_list = System.getenv("DNS_PATH")
-val feedback_file = "None"
+val feedback_file = System.getenv("LPATH") + "/dns_scores.csv"
 val duplication_factor = 100
```

Comment:
at present, this should be read from the System.getenv("DUPFACTOR")   per the code in 1.0.1 version of ml_ops.sh

in the near future, I'd like to change it to a command line argument,  but that's for another checkin, not today